core/SConscript.unix

```diff
@@ -21,7 +21,7 @@ if BENCHMARK and PYOPT != '0':
   print("BENCHMARK=1 works only with PYOPT=0.")
   exit(1)
 
-FEATURES_WANTED = ["input", "sd_card", "dma2d", "optiga"]
+FEATURES_WANTED = ["input", "sd_card", "dma2d", "optiga", "tropic"]
```

Comment:
firmware and kernel should also have this, right?

Reply:
I think this has been answered in https://github.com/trezor/trezor-firmware/pull/4172#discussion_r1977775557.